core/pl/geohash.pm.sdoc

```diff
@@ -104,4 +104,15 @@ if (1 << 32) {
 *ghe = \&geohash_encode;
 *ghd = \&geohash_decode;
 
+*gh_box = sub {
```

Comment:
it's a bit subtle.

the code above has to predicate on the width of perl ints -- so we don't know at compile-time which version of the function we'll want. that means we BEGIN/assign instead of using `sub{}`.

in your case you could use `sub gh_box {}` (and that would be idiomatic) because the definition is always the same.